src/main/java/br/com/mercadolivre/bootcampw2/grupo11/socialmeli/entities/Post.java

```diff
@@ -0,0 +1,31 @@
+package br.com.mercadolivre.bootcampw2.grupo11.socialmeli.entities;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.util.Date;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@Entity
+@Table(name = "post")
+public class Post {
+    private Long userId;
```

Comment:
You dont need this field anymore, because you already have the relationship declared in the "seller" field